gatsby/gatsby-config.js

```diff
@@ -1,4 +1,6 @@
-const stylelint = require('stylelint');
+// const reporter = require('postcss-reporter');
```

Comment:
Are we commenting this out because we plan on revisting its usage?

Reply:
That is correct.